src/tools/clippy/tests/ui/implicit_return.stderr

```diff
@@ -8,8 +8,7 @@ LL |     true
    = help: to override `-D warnings` add `#[allow(clippy::implicit_return)]`
 help: add `return` as shown
    |
-LL -     true
-LL +     return true
+LL |     return true
```

Comment:
this renders strangely since it has no addition `+`'s, but it's exactly how it rendered before #127541, so it's a preexisting bug in the emitter.